TowerForge/TowerForge/Rendering/Renderer.swift

```diff
@@ -12,13 +12,19 @@ class Renderer {
     private unowned var target: Renderable
     private unowned var scene: SKScene?
 
-    private var renderedNodes: [UUID: TFAnimatableNode] = [:]
+    private var renderedNodes: [UUID: TFSpriteNode] = [:]
 
     init(target: Renderable, scene: GameScene?) {
         self.target = target
         self.scene = scene
     }
-
+    func addNodeToScene(entity: TFEntity) {
+        guard let spriteComponent = entity.component(ofType: SpriteComponent.self) else {
+            return
+        }
+        renderedNodes[entity.id] = spriteComponent.node
+        self.scene?.addChild(spriteComponent.node)
+    }
```

Comment:
What's the purpose for this? This seems to only be used by Point but it is already added to the EntityManager so it should be rendered by the render method.

Reply:
changed back